### PR TITLE
adds about copy and email address to about section

### PR DESCRIPTION
**Pull request by ilovevinylrecs**
here's my first pull request

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+      Previously he was a Director at Insound.com / Warner Music Group as well as
+      a Director of Audience Growth, Ad Operations, E-commerce, and Digital
+      Marketing at The FADER.<br />
+      Andy is mostly obsessed with music and vinyl records (all genres, mostly
+      1980 - today), alternative movie posters and pop art, comic books and
+      graphic novels, and coding.<br />
+      Originally from Long Island, NY, he now lives in Long Island City, NY.
+    </p>
+    <p>Email:
+    <a href="mailto: andrew.nystrom@gmail.com" target="_blank"
```
Comment (desmondmorris) [inline about.html]: Can you remove the space after the colon?

File: about.html [hunk 2]
```diff
@@ -1,7 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
   <head> </head>
-  <body>
-    Hi I'm Andy.
+  <body style="font-family: monospace">
+    </p>Andy Nystrom is a husband and father to an 8 year old yellow lab mix named
```
Comment (desmondmorris) [inline about.html]: I think this should be an open <p> tag

**Review by desmondmorris — CHANGES_REQUESTED** [review]
Looking good, just a few suggestions

File: about.html (hunk 1 above)
Comment (ilovevinylrecs) [reply about.html]: thanks for catching, removed space.

File: about.html (hunk 2 above)
Comment (ilovevinylrecs) [reply about.html]: yes, changed this to open tag.

**Pull request merged**